CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * The default template alias `builtin_log_root(change_id: ChangeId, commit_id: CommitId)` was replaced by `format_root_commit(root: Commit)`.
 
+* When targeting Windows, libssh2 will use OpenSSL rather than wincng. This fixes
+  interacting with Git+SSH hosts using ED25519 keys, but Windows developers will
+  need to provide OpenSSL at build time. This is enabled by Cargo feature
+  `openssl-on-win32`.
```

Comment:
Do Windows developers actually need to do anything in particular? The branch builds out of the box for me on win10 - libssh2-sys does the vendoring for us.

Reply:
Not anymore. I hadn't updated the CHANGELOG yet, oops